src/services/posthog/PostHogClientProvider.ts

```diff
@@ -48,18 +48,16 @@ export class PostHogClientProvider {
 			this.telemetrySettings.host = isTelemetryEnabled
 		})
 
-		if (vscode?.env?.isTelemetryEnabled === false) {
-			this.telemetrySettings.host = false
-		}
-
-		const config = vscode.workspace.getConfiguration("cline")
-		if (config.get("telemetrySetting") === "disabled") {
-			this.telemetrySettings.cline = false
```

Comment:
when we check isTelementryEnabled we checked `this.telemetrySettings.cline && this.telemetrySettings.host`
Since we removed `vscode.workspace.getConfiguration("cline")` and not setting it anymore, telementry is going to be off until we add the global settings for cline telemetry check back in